kubectl/private-network-clique/configmap/configmap.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: besu-configmap
+  labels:
+    app: besu-configmap
+data:
+  validator1PubKey: "5d812c3c25ff398ab416968fce9009c2be7ed70a87abc8ea30bd667ce17a9287a6341fbf6ce757bb8148436c39c71296639ea81afcc94cdf908b6e1344f26188"
+  validator2PubKey: "b2fba529681ea7f4619556753d40c8689b936fb1c621bc91f94d2938eb58c285d4911457ae4887b9c3bd593b2d608d319c6dc384d6acae2d043a4657029178d3"
+  validator3PubKey: "00b20ab6a385a2403d64637b3d93cb6d83215a08f29adb6feb4b8bf03387b734444e8b060f53150dea4b9b897823540d19918c13d6f57a5153d190b5fad7bf51"
+  validator4PubKey: "6b4c9ccef1d84e7d86814eee56543fea05bbbad73e398ce31ad95b9aaafd0072494ddfff20a5a6b47cc111f40cf3204793e7c6fbeae0f5854564900523536fa7"
+  nodesHttpCorsOrigins: "*"
+  nodesHostWhitelist: "*"
+  genesis.json: |-
+    {
+      "config":{
+        "chainId": 1729,
+        "constantinoplefixblock": 0,
+        "clique":{
+          "blockperiodseconds":5,
+          "epochlength":30000
+        }
+      },
+      "coinbase":"0x0000000000000000000000000000000000000000",
+      "difficulty":"0x1",
+      "extraData":"0x0000000000000000000000000000000000000000000000000000000000000000<signers>0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Hi @ppoliani could you update this to contain the addresses of the signers please? Idea being we'd like to get people going with the config in this template to see it working and we mention in the readme that they have to make their own keys, config etc for production.